spectrum/src/main/java/com/thebluealliance/spectrum/internal/SpectrumPreferenceDialogFragmentCompat.java

```diff
@@ -14,7 +14,9 @@
 public class SpectrumPreferenceDialogFragmentCompat extends PreferenceDialogFragmentCompat {
 
     private SpectrumPalette mColorPalette;
-    private @ColorInt int mCurrentValue;
+    @ColorInt
```

Comment:
My convention is to leave single annotations on the same line as the variable they apply to.